retrofit-adapters/rxjava2/src/test/java/retrofit2/adapter/rxjava2/FlowableTest.java

```diff
@@ -136,14 +136,14 @@ public void responseFailure() {
   public void responseRespectsBackpressure() {
```

Comment:
These tests can all be removed, and I suspect there's code in the implementation of the adapter that can also be removed if an initial 0 request is no longer valid.

Reply:
I meant the individual test cases. Not the whole file.

But like I said there's larger changes needed anyway.